modules/cnf-topology-aware-lifecycle-operator-preparing-for-updates.adoc

```diff
@@ -6,51 +6,118 @@
 [id="talo-platform-prepare-end-to-end_{context}"]
 = End-to-end procedures for updating clusters in a disconnected environment
```

Comment:
This section and its contents should be moved out one level instead of under "Updating managed policies with the Topology Aware Lifecycle Operator".